src/stubber/codemod/_partials/db_entry.py

```diff
@@ -1,19 +1,32 @@
-from typing import TYPE_CHECKING, type_check_only
+"""
+This file contains the partials that are used to generate the DB variant of createstubs.py
+
+- type_check_only is used to avoid circular imports
+
+The partial in enclused in ###PARTIAL### and ###PARTIALEND### markers
+i
+"""
```

Comment:
@Josverl

My idea behind this approach was for ease of use when creating / modifying the code mods.

I figured this pattern was much easier to maintain than the string blocks used in my first approach:
https://github.com/Josverl/micropython-stubber/blob/abc2c68186c8a46c06b02535d57790cd330a4655/src/stubber/codemod/board.py#L94-L108

That became pretty messy when I tried to apply it to this much larger modification.

The `@type_check_only` decorator here is not strictly to “avoid circular imports” but is to actually avoid any imports at all. This is because this file is never meant to actually be imported by any file; only read and exposed by the Partial enum:

https://github.com/Josverl/micropython-stubber/blob/abc2c68186c8a46c06b02535d57790cd330a4655/src/stubber/codemod/_partials/__init__.py#L26-L34